### ISSUE TITLE
chaosblade-operator 1.5.0 版本 k8s下发网络故障失败

### ISSUE THREAD

**Issue by hengyu-coder** [chaosblade-io/chaosblade-operator#137]
## Issue Description

chaosblade-operator 1.5.0 版本 k8s下发网络故障失败，默认走了目标容器内执行的tc命令导致下发失败
通过定位代码发现新版本修改了容器网络故障命令生成判断条件
我修改的实现代码本地自测通过；准备提高一个PR

**Comment by hengyu-coder** [comment]
fix 137 issues pr

**Comment by lss9981** [comment]
升级到1.5.0后，我也发现了pod级别的网络故障失败问题
[https://github.com/chaosblade-io/chaosblade/issues/665](url)

**Comment by hengyu-coder** [comment]
我排查到问题了， 提交了PR

**Comment by lss9981** [comment]
> 我排查到问题了， 提交了PR

你的提交好像check fail了

**Comment by hengyu-coder** [comment]
@lss9981 没看到check fail具体原因，这个应该是代码提交规范检测不通过吧

**Comment by lss9981** [comment]
> @lss9981 没看到check fail具体原因，这个应该是代码提交规范检测不通过吧

是的 [https://github.com/chaosblade-io/chaosblade-operator/pull/138/checks?check_run_id=5299528260](url)

**Comment by oss-sentinel-ai** [comment]
## Issue Evaluation

**Category:** `type/bug` | **Status:** Confirmed

### Problem Analysis

The network fault experiment fails in chaosblade-operator 1.5.0 because the `tc` command is executed inside the target container instead of on the host node. This is a regression from previous versions.

### Root Cause

The issue reporter identified that version 1.5.0 modified the condition for determining where to execute the `tc` command. The network experiment should execute `tc` on the host node's network namespace, not inside the container.

The code change in 1.5.0 likely altered the logic in `exec/model/executor_nsexec.go` or `exec/model/executor_copy.go` that determines whether to use `nsenter` to enter the host network namespace.

### Impact
- **Severity:** High — all network experiments fail in 1.5.0
- **Affected:** `k8s pod-network` experiments (delay, loss, duplicate, corrupt, reorder)
- **Versions:** chaosblade-operator 1.5.0

### Note

The issue reporter mentioned they have a fix and plan to submit a PR. If a PR is submitted, it should be reviewed to ensure:
1. The `tc` command executes in the correct network namespace
2. The fix doesn't break other experiment types
3. Backward compatibility with previous versions is maintained

---
*Automated evaluation by oss-sentinel-ai*
